Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -5,10 +5,6 @@
     @apply pb-0;
   }
 
-  &--bg-white {
-    @apply bg-white;
-  }
-
   &--dark {
```

Comment:
`section--dark` orange title and white text is overridden by defaults from the classes below.
Please arrange CSS for specialty classes to be defined after the base ones, and for them to have more CSS specificity.

![image](https://user-images.githubusercontent.com/2000200/151560946-b523e83f-b98c-4b52-9bf2-4d921bb185c5.png)